MsmqReturnToSourceQueue/ErrorManager.cs

```diff
@@ -140,13 +164,4 @@ string GetOriginalId(Dictionary<string, string> headers)
 
         return null;
     }
-
-
-    const string NoMessageFoundErrorFormat = "INFO: No message found with ID '{0}'. Checking headers of all messages.";
```

Comment:
if we ever intend to submit our changes back to the original repo, moving declarations to suit a different style would be considered bad form:
http://tirania.org/blog/archive/2010/Dec-31.html
> When you contribute fixes or new features to an open source project you should use the existing coding style, the existing coding patterns and stick by the active maintainer's choice for his code organization.